includes/cli/backfillers/class-reader-registered.php

```diff
@@ -29,10 +31,14 @@ protected function get_processed_item_output( $event ) {
 	 * @return \Newspack_Network\Incoming_Events\Abstract_Incoming_Event[] $events An array of events.
 	 */
 	public function get_events() {
+		$roles_to_sync = \Newspack_Network\Utils\Users::get_synced_user_roles();
+		if ( $roles_to_sync === [] ) {
```

Comment:
Nit: Maybe we can use something like PHP `empty` here

Reply:
Done in 121cc686236bac0458e5a8e811fa2380fca29171